main.go

```diff
@@ -158,6 +162,9 @@ func parseArgs(a *app) func() int {
 		return func() int { return 1 }
 	}
 
+	if versionCmd.Happened() {
+		fmt.Printf("pgCarpenter version %s (git: %s)\n", version, gitCommit)
```

Comment:
should this exit the program?

Reply:
We should exit here. If we let function exit on its own, exit status will be 1 which is not what we want